spec/std/int_spec.cr

```diff
@@ -2,11 +2,11 @@ require "spec"
 require "big"
 
 private def to_s_with_io(num)
-  String.build { |str| num.to_s(str) }
+  String.build { |io| num.to_s(io) }
```

Comment:
Unrelated change. I also prefer the `str` idiom for `String.build`. We're building a string, not whatever io.

Reply:
I can revert that.

But I always find this weird ("why is this passing a String?"): `str` is not a string but a string builder (so it would make sense to call it `builder`). Which is an IO and `io` is shorter but equally expressive. The argument in the method definition of `to_s` is also called `io`. So it makes very much sense to call this `io`.